crates/backend/src/codegen.rs

```diff
@@ -798,7 +834,10 @@ impl TryToTokens for ast::Export {
                     export_name = #export_name,
                 )]
                 pub unsafe extern "C" fn #generated_name(#(#args),*) -> #wasm_bindgen::convert::WasmRet<#projection::Abi> {
-                    #start_check
+                    #[automatically_derived]
+                    const _: () = {
```

Comment:
```suggestion
                    const _: () = {
```
`#[automatically_derived]` only works on `impl` blocks, it has no effect when placed over any other item. But `const _` already has the same effect.

We (I) might have been overzealous and wrongly applying this in the past as well.